### DISCUSSION TITLE
Advice to fix my screwed up versioning

### DISCUSSION THREAD

**Discussion by fwawrosz** [GitTools/GitVersion#2971] [Q&A] [answered]
Hello Community,

This is the situation:
I have my master branch which is at version 5.x.x
I have a Releases/v4.0 branch which is supposed to be at version 4.x.x but at some point the version increased to 5.x.x which I do not want. 
I need the master branch to be at version 5.x.x and Releases/v4.0 to be at version 4.x.x. 

Any advice on how to get the Releases/v4.0 branch back to version 4.x.x ?

Thanks in advance.
Felix

**Comment by asbjornu** [comment]
You can try to set [`track-merge-target`](https://gitversion.net/docs/reference/configuration#track-merge-target) to `false` for  your `release` branch.

**Reply by HHobeck** [reply, marked as answer]
You cannot have both. You need to ignore the commit message in the configuration or remove the tag which is repsonsible for 5.x.x. If you go with configuration I would say have one configuration for main and one configuration for the rest with ignoring setup.

Happy Branching!
